### DISCUSSION TITLE
What is better for sound meaning ease, quality, stability,compatability with chan-quectel, EC25 or imcom 7600 ?

### DISCUSSION THREAD

**Discussion by CpServiceSpb** [RoEdAl/asterisk-chan-quectel#8] [General]
I use EC25-E.
And look to side of Simcom 7600G-h, because it supports more bands.

What is better for sound meaning уфыуб quality, stability,compatability with quectel channel, EC25 or imcom 7600 ?

**Comment by RoEdAl** [comment]
Because I can't turn on *VoLTE* on *Quectel* or *SimCom* module there's no difference of sound quality for me. From developer's perspective *SimCom* module seems to be more stable. The Quectel module sometimes gets stuck and the only way to unblock it is to turn the power off and and then on again. Maybe this may be fixed by firmware upgrade but Quectel's support is very limited.

Quite good sound quality may be achieved using old *SimCom* [SIM900](https://github.com/RoEdAl/asterisk-chan-quectel/wiki/Using-SIM900-board-with-analog-input-and-output) or SIM800 modules with **analog** input and output. But this solution requires more work (additional sound card, additional wiring, etc.) and you are limited to 2G only.

**Comment by CpServiceSpb** [comment]
Thans for the answering.
What about Sim7600G-H ?
About 2G/3G sound quality, stability and compatability in these moes.
And can reccomend reliable Aliextress seller for Simcom at all and mPCIE or m2 Sim7600G-H in particular ?
